src/ContentAdapter.php

```diff
@@ -174,18 +174,12 @@ public function putStream($path, $resource, Config $config): Document
     /**
      * @param string $path
      *
-     * @return string
      * @throws Exception
      */
-    public function read($path): string
+    public function read($path)
```

Comment:
Zašto nemamo više return type?

Iako nije u ovom PRu, `write()` metoda ne valja, `delete() rename() copy()`...i ostali...imaju krivi return type i dalje